api/serverpb/server.proto

```diff
@@ -146,6 +120,8 @@ message STTCheckIntervals {
 
 // Settings represents PMM Server settings.
 message Settings {
+  reserved 13, 14; // Removed fields
```

Comment:
I guess we can just remove them now, and reorder all fields. Or that can be done in separate ticket for the whole API.

Reply:
It's not the client, I also think we can go ahead and just delete.

Reply:
Let's do this in the scope of whole API review